### DISCUSSION TITLE
Fans not settling down again automatically after gaming

### DISCUSSION THREAD

**Discussion by solidfyre** [seerge/g-helper#2774] [Q&A]
I am not sure if this is a bug or not, but for some reason my fans never settle down after gaming.

When I start a game the fans ramp up normally but they never settle down afterwards and continues to run at high RPM indefinitely. 
It's been like this for months so I am not sure what version of G-helper introduced this problem, but it used to work before.
What I do is go to silent mode and back to balanced quickly and they instantly starts to settle down. It's like it resets the sensor somehow.

I am running the Asus Zephyrus M16 GU604VZ, newly installed Windows 11 with G-helper only. No Asus services running. Control Interface installed of course. I haven't really changed much in G-Helper, Windows Power Mode on "Best Performance" and disable CPU Boost.

Anyone else experience this?

![image](https://github.com/seerge/g-helper/assets/129062438/1afed90e-7543-40e2-a3b6-0bdb28669217)

**Comment by seerge** [comment]
@solidfyre hello, 

Fans are [always controlled by BIOS ](https://github.com/seerge/g-helper/wiki/FAQ#how-does-g-helper-control-my-fan-speeds). 

If you don't like how default fan curves behave, you can try to apply a custom ones :)

Thanks

**Reply by solidfyre** [reply]
Thanks for the clarification Seerge.
So I guess it would be safe to assume it's a bug in the bios then. Afaik the fans are supposed to follow sensor output data. CPU temp goes up, fans spins faster etc, but it seems it doesn't go the other way, unless I actively change mode once back and forth in G-Helper. hmm.

**Reply by seerge** [reply]
@solidfyre it could be :) If you want you can always try [Experimental build with manual fan control](https://github.com/seerge/g-helper/discussions/2272). There app will try to set fan speed in realtime.

**Reply by solidfyre** [reply]
Hey @seerge, I know you said fans are controlled by bios, but did you change something recently? 
With 0.178 the fans now settle down automatically after I quit a game, as expected (yay🎉). :)
I read the change logs but couldn't find anything specific about it. Afaik I haven't updated anything else.

**Reply by seerge** [reply]
@solidfyre i didn't change anything. Keep in mind that if you use custom settings on top of default profile (i.e. power limits / fan curves) behavior may be different. So you may want to click `Factory Defaults` and use default profile instead.

P.S. I have miss-read ... they do settle down . Good for you :)

**Reply by solidfyre** [reply]
Yeah, I am running all defaults, no custom fan curves or anything. 
Anyway, its working as expected. Thanks for G-helper, it makes my computer actually usable. I sent you a dinner ;)

**Reply by seerge** [reply]
@solidfyre great, thanks
